tests/src/test/scala/cats/tests/TryTests.scala

```diff
@@ -25,6 +24,9 @@ class TryTests extends CatsSuite {
   checkAll("Try", MonadTests[Try].monad[Int, Int, Int])
   checkAll("Monad[Try]", SerializableTests.serializable(Monad[Try]))
 
+  checkAll("Try[Int]", ApplicativeEvalTests[Try].applicativeEvalWithError[Int, Int, Int])
+  checkAll("ApplicativeEval[Try]", SerializableTests.serializable(ApplicativeEval[Try]))
```

Comment:
I also added this law (notice `applicativeEvalWithError`):

```scala
eval(always(throw ex)) == raiseError(ex)
```

It's an *optional* law that happens in case `F[_]` also has an `ApplicativeError[A, Throwable]`. Mentioning this because `Try` follows this law, whereas `Eval` does not.